libcontainer/cgroups/fs/devices.go

```diff
@@ -11,6 +11,12 @@ import (
 type DevicesGroup struct {
 }
 
+type CgroupParams struct {
+	initpid int
+}
+
+var cg CgroupParams
```

Comment:
Does this cause issues if you have lots of load on the machine? Having a global variable for this seems like a bad idea IMO.